srv/GraspPlanning.srv

```diff
@@ -0,0 +1,27 @@
+# Requests that grasp planning be performed for the target
+# returns a list of grasps to be tested and executed
+
+# the planning group used
+string group_name
+
+# the object to be grasped
+CollisionObject target
```

Comment:
The name is taken from the original `manipulation_msgs/GraspPlanning.srv`.
I prefer the short name here.